core/modules/wsched/ChunkDisk.cc

```diff
@@ -89,7 +89,7 @@ wbase::Task::Ptr ChunkDisk::MinHeap::pop() {
 }
 
 
-void ChunkDisk::enqueue(wbase::Task::Ptr const& a) {
+void ChunkDisk::queTask(wbase::Task::Ptr const& a) {
```

Comment:
There are several other functions named queCmd, queTask, etc. I've been using 'que' as flag to indicate that the function is placing a Command on an EventThread in hopes that it is clear that something is being scheduled to happen later, likely on a different thread.